### PR TITLE
Add A Responsive Contextual Menu To Admin

### PR DESCRIPTION
**Pull request by MatthewKennedy**
An idea to make the admin more usable on mobile devices.

Screenshot 2020-06-25 at 11 57 11

Screenshot 2020-06-25 at 11 57 02

### CODE REVIEW

File: backend/app/assets/javascripts/spree/backend/admin.js [hunk 1]
```diff
@@ -15,29 +15,50 @@ jQuery(function ($) {
...
-  // Off Canvas Sidebar Functionality
+  // Responsive Menus
+  var body  = $('body')
+  var modalBackdrop  = $('#multi-backdrop')
+
+  // Fail safe on resize
+  document.getElementsByTagName("BODY")[0].onresize = function() { closeAllMenus() }
+
+  function closeAllMenus() {
+    $(body).removeClass()
```
Comment (AleksandraKaminska) [inline backend/app/assets/javascripts/spree/backend/admin.js:26]:
```suggestion
    body.removeClass()
```

**Review by AleksandraKaminska — APPROVED** [review]

**Review by damianlegawiec — APPROVED** [review]
Thank you @MatthewKennedy - this is a great addition!

**Comment by MatthewKennedy** [comment]
@damianlegawiec If you're all happy with this and it makes the cut, It gives a good base to move forward and work through the admin UI looking for occurrences where the content is not fully responsive and causes unnecessary side scroll.

Do you want me to do that on this PR or wait and open a new PR if this make the master?

**Comment by damianlegawiec** [comment]
> @damianlegawiec If you're all happy with this and it makes the cut, it gives a good base to move forward and work through the admin UI looking for occurrences where the content is not fully responsive and causes unnecessary side scroll.
> 
> Do you want me to do that on this PR or wait and open a new PR if this one make it into the master?

We'll merge this one into master now and you can create sub-sequent PRs after that, thank you!

**Comment by damianlegawiec** [comment]
Refers https://github.com/spree/spree/issues/8287

**Pull request merged**